### DISCUSSION TITLE
Is possible self mount on subpath?

### DISCUSSION THREAD

**Discussion by jmtt89** [tolgee/tolgee-platform#1207] [Q&A] [answered]
I trying to self hosting tolgee, i put on subpath something like https://myserver/tolgee but returns error when trying to fetch css and js files.

i check on chorme console and is seeking on 
- https://myserver/css/... 
- https://myserver/js/...

i doublecheck on documentation https://tolgee.io/docs/platform/self_hosting/configuration but not see any property for set a different path

The first look the docs i was think tolgee.front-end-url do the job, but i set to https://myserver/tolgee and nothings happened :(

**Comment by JanCizmar** [comment, marked as answer]
Hi there! I don't think this would be ever possible. You have to host it using different subdomain.

**Reply by jmtt89** [reply]
oh alright, very thanks. Subdomain will be!!
